contracts/id.tmy/include/id.tmy/id.tmy.hpp

```diff
@@ -145,6 +151,16 @@ namespace idtmy
       using updatekeyper_action = action_wrapper<"updatekeyper"_n, &id::updatekeyper>;
       using linkauth_action = action_wrapper<"linkauth"_n, &id::linkauth>;
 
+      struct [[eosio::table]] account_type {
+         name account_name; 
+         enum_account_type account_type; 
+
```

Comment:
should add a version variable here, and we can remove it from the app and people table